lib/commands.js

```diff
@@ -596,11 +596,11 @@ var operations = module.exports.operations = {
 
   confirmDelete: function(context) {
     if (context.overrides.force) return context.next();
-
-    prompt.confirm('Are you sure you want to delete ' + context.stackName + '?', function(err, ready) {
+    var msg = 'Are you sure you want to delete ' + context.stackName + ' in region ' + context.stackRegion + '?';
+    prompt.confirm(msg, function(err, ready) {
       if (!ready) return context.abort();
       context.next();
-    });
+    }, false);
```

Comment:
We should put this optional value before the callback, to conform with the expected javascript callback pattern.